examples/scenarioSmallBodyLandmarks.py

```diff
@@ -0,0 +1,571 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+r"""
+Overview
```

Comment:
This script doesn't run on my machine.  I'm getting error messages.

Reply:
My bad. I did a modification to gravCoeffOps.py to support another type of polyhedron file (as it is the one used in the script) but forgot to commit it. Being the landmark distribution based on the previous file and it does not load properly, god knows what is happening

I've committed the my change to gravCoeffOps.py, let me know if there is more trouble

Reply:
Cool, the scenario scripts now runs.  Nice job with the 3D asteroid figures showing what landmarks are visible at two different time frames.